sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobAsyncClient.java

```diff
@@ -236,6 +244,140 @@ public void cancel(Context context, long transferId) {
             .cancel(transferId);
     }
 
+    /**
+     * Creates a new container within a storage account. If a container with the same name already exists, the operation
+     * fails.
+     *
+     * @param containerName The container name.
+     * @param callback      Callback that receives the response.
+     */
+    public void createContainer(String containerName,
```

Comment:
I don't know why the PR includes all a ton of other things, I guess I should just ignore them? I assume this PR only introduces the `setBlobMetadata` operations, correct?